omod/src/main/webapp/resources/scripts/order-entry.js

```diff
@@ -229,14 +229,16 @@ angular.module("orderEntry", ['orderService', 'encounterService', 'session'])
                     delete transformed.editing;
                     return transformed;
                 });
+                
                 var encounter = {
                     patient: encounterContext.patient.uuid,
                     encounterType: encounterContext.encounterType.uuid,
                     visit: uuidIfNotNull(encounterContext.visit),
                     location: uuidIfNotNull(encounterContext.location),
-                    provider: provider.person.uuid, // submit the person because of RESTWS-443
+                    // provider: provider.person.uuid, // submit the person because of RESTWS-443
```

Comment:
I have considered comments and added encounterProviders property. I read the description of RESTWS-443 issue and it was mentioned that encounterProviders is an array of objects and each object has two properties 1. provider 2. encounterRole

{
    ...
    "encounterProviders": [
        { "provider": "uuid-of-provider", "encounterRole": "uuid-of-encounter-role" }
    ]
}

I did accordingly but i get an error and here it is https://talk.openmrs.org/t/options-for-openmrs-drug-management/1719/40